content/docs/tooling/cli/_index.md

```diff
@@ -59,6 +61,481 @@ Outputs help text
+            colocatedRemote:
+              nodeSelectorTerms:
+                - matchExpressions:
+                  - key: "topology.kubernetes.io/zone"
+                    operator: "In"
+                    values:
+                      - "zone-b"
+            colocatedBoth:
+              nodeSelectorTerms:
+                - matchExpressions:
+                    - key: "topology.kubernetes.io/zone"
+                      operator: "In"
+                      values:
+                        - "zone-ab"
+  - endpoint: 10.0.0.2
+    username: user
+    metro-replication:
+      - hostConnectivity:
+          local:
+            nodeSelectorTerms:
+              - matchExpressions:
+                - key: "topology.kubernetes.io/zone"
+                  operator: "NotIn"
+                  values:
+                    - "zone-a"
+                    - "zone-b"
+                    - "zone-ab"
+          metro:
+            colocatedLocal:
+              nodeSelectorTerms:
+                - matchExpressions:
+                  - key: "topology.kubernetes.io/zone"
+                    operator: "In"
+                    values:
+                      - "zone-a"
+            colocatedRemote:
+              nodeSelectorTerms:
+                - matchExpressions:
+                  - key: "topology.kubernetes.io/zone"
+                    operator: "In"
+                    values:
+                      - "zone-b"
+            colocatedBoth:
+              nodeSelectorTerms:
+                - matchExpressions:
+                    - key: "topology.kubernetes.io/zone"
+                      operator: "In"
+                      values:
+                        - "zone-ab"
+
+replication:
+  sourceClusterID: "cluster-1"
+  targetClusterID: "cluster-2"
+  parameters:
+    rpo: "Five_Minutes"
+    mode: "ASYNC"
+    ignoreNamespaces: false
+    volumeGroupPrefix: "csi"
+```
+{{< /collapse >}}
+{{< collapse id="dellctl-install-powerstore-full-config" title="Configuration with all available storage parameters" card="false" >}}
+```yaml
+# Global Driver parameters
+namespace: powerstore
+config-version: {{VERSION}}
+csi-volume-prefix: myvol
+csi-node-prefix: nodepre
+operator-install: true
+machineconfig: true
+snapshot-controller: true
+validate-connectivity: true
+registry-url: my.registry.com:5000/dell/csm
+modules: replication,authorization,observability,resiliency
+ 
+# Parameters for each PowerStore system
+storage:
+  - endpoint: 10.0.0.1
+    username: user
+    block-protocol: FC
+    nfs-acls: 0777
+    skip-certificate-validation: true
+    primary: true
+    storage-class:
+      - fsType: ext4
+        reclaimPolicy: Delete
+        volumeBindingMode: WaitForFirstConsumer
+        allowVolumeExpansion: true
+        allowedTopologies:
+          - matchLabelExpressions:
+              - key: csi-powerstore.dellemc.com/10.0.0.1-fc
+                values:
+                  - true
+    metro-replication:
```

Comment:
In the 'all available parameters' section, we can remove primary/secondary and metro-replication parameters since we have specific examples above.